packages/models-library/src/models_library/projects_nodes.py

```diff
@@ -164,7 +164,7 @@ class Node(BaseModel):
             ge=0,
             le=100,
             description="the node progress value (deprecated in DB, still used for API only)",
-            deprecated=True,
+            deprecated=True,  # <-- I think this is not true, it is still used by the File Picker (frontend node)
         ),
```

Comment:
I think this might be the case for frontend nodes. but these should be moved to the UI part. creating an issue about this would be good.